### ISSUE TITLE
VoiceOver does not speak the keys as you type them

### ISSUE THREAD

**Issue by mrgrieves4** [allisonfm85/AccessTerm#14]
I have VoiceOver set to speak characters and words as I type them. However, in AccessTerm when I am typing in a command, this does not work, so I don’t hear anything unless I arrow back through what I’ve typed. As I am not very good at typing, it’s handy to know that what I think I have typed is actually what my fat fingers have done.

I am using Tahoe 26.6.2

Thanks!
